packages/composables/src/internalHelpers/defaultApiParams.json

```diff
@@ -3,26 +3,25 @@
     "limit": 10,
     "associations": {
       "manufacturer": {},
-      "properties": {
-        "associations": {
-          "group": {}
-        }
-      },
       "media": {},
       "productReviews": {},
-      "children": {
-        "associations": {
-          "options": {
-            "associations": {
-              "productOptions": {},
-              "group": {}
+        "crossSellings": {
```

Comment:
Wrong indentation.

This way it also fetches the crosssellings when listing pages are loaded? Will ultimately have a major performance impact. IMO we should remove this part from the initial page resolver request and do it asynchronously in the future. Wdyt?

Reply:
right!

I've created an issue for that: https://github.com/vuestorefront/shopware-pwa/issues/1305